.github/actions/repeat/action.yml

```diff
@@ -52,8 +55,13 @@ runs:
         else
           FORCE_EPOLL=""
         fi
-        echo Running command: timeout ${{ inputs.timeout }} pytest ${{ inputs.expression }} --drop-data-after-each-test ${FORCE_EPOLL} --color=yes --json-report --json-report-file=report.json --log-cli-level=DEBUG --count=${{ inputs.count }}
-        timeout ${{ inputs.timeout }} pytest ${{ inputs.expression }} --drop-data-after-each-test ${FORCE_EPOLL} --color=yes --json-report --json-report-file=report.json --log-cli-level=DEBUG --count=${{ inputs.count }} || code=$?
+        if [[ $"{{ inputs.vmodule_expression }}" != "" ]]; then
+          VMOD="--df vmodule=${{ inputs.vmodule_expression }}"
```

Comment:
isn't it `--vmodule` instead of `--df vmodule` ?

Reply:
Yes for dragonfly the arg should be `--vmodule`.

This arg however is passed to the pytest arg parser, where it will be converted into the correct format before being passed to dragonfly https://github.com/dragonflydb/dragonfly/blob/3b5628dd327591c19ba678bcd794f580146869f6/tests/dragonfly/instance.py#L366